lib/flagsmith-es/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "flagsmith-es",
-  "version": "3.16.0",
+  "version": "3.18.0",
```

Comment:
Is there a reason we're skipping 3.17.0 here?

Reply:
Ah, I thought there was a 3.17 but it turns out all of these were beta releases. I won't be able to adjust this now.

Reply:
All good.